subgraph/.eslintrc.yml

```diff
@@ -7,3 +7,6 @@ ignorePatterns:
   - imported/
   - generated/
   - extended-schema.ts
+  - tests/
+
+rules: {'@typescript-eslint/ban-types': ['error', {'types': {'BigInt': false}}]}
```

Comment:
The idea is that we have a global linting configuration that we apply everywhere. Should this be in here only? In https://github.com/aragon/osx/blob/develop/packages/subgraph/.eslintrc.json, we don't seem to have this rule and I am wondering why it is needed here

Reply:
Yes but this rule should be only applied to subgraph because eslint detects the word `BigInt` as the native JS class an throws an error saying that wee should use `bigint` instead, however we want to use `BigInt`because it is not the the one native to JS is the one imported from matchstick